bin/catkin_create_pkg

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
It is unclear to me if this is the right / recommended thing to do (I'm looking primarily at https://peps.python.org/pep-0394/#for-python-script-publishers) which includes

> For scripts that are only expected to be run in an activated virtual environment, shebang lines can be written as `#!/usr/bin/env python`, as this instructs the script to respect the active virtual environment.
> In cases where the script is expected to be executed outside virtual environments, developers will need to be aware of the following discrepancies across platforms and installation methods:
> 
> * Older Linux distributions will provide a `python` command that refers to Python 2, and will likely not provide a `python2` command.
> * Some newer Linux distributions will provide a `python` command that refers to Python 3.
> * Some Linux distributions will not provide a `python` command at all by default, but will provide a `python3` command by default.

I think it depends on which platforms we expect these scripts to be run. I have pretty much talked myself into preferring the `python3` since it very explicitly breaks python2 support which we are very explicitly breaking.

IMO we have latitude here to do either since the "most right" answer is probably to use entry points which will install the scripts appropriately but we're not making that change today.

Reply:
I'm pretty sure that console_scripts entry points are installing the executables we see in the packages, but I think the scripts are still used when running tests.

I don't feel strongly. Throw me a :+1: on this comment and I'll revert it.